src/main/java/duke/parser/DateParser.java

```diff
@@ -0,0 +1,167 @@
+package duke.parser;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import duke.exception.InvalidDateInputException;
+
+/**
+ * Represents a Date Parser to parse in all String data given into a form readable by the user.
+ */
+public class DateParser {
+
+    private SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yyyy hhmm");
+    private Date inputParser;
+    private String input;
+    private Calendar calendar = Calendar.getInstance();
+
+    /**
+     * Reads the input and stores it in the parser.
+     * @param input The input provided by the user.
+     */
+    public void readInput(String input) {
+        this.input = input;
+    }
+
+    /**
+     * Reads the month (provided an integer) and returns it as a String.
+     * For example, 0 represents January, 1 represents February all the way until 11 represents December.
+     * @param month The integer representation of the month.
+     * @return The String representation of the month.
+     */
+    public String monthToString(int month) {
+        if (month == 0) {
```

Comment:
You might want to use switch statements instead?

Reply:
Hi Yi Yin!

This is a personal preference by me actually to use if else statements! 

Thanks though for the comment!